docusaurus.config.ts

```diff
@@ -127,7 +127,7 @@ const config: Config = {
     },
     announcementBar: {
       id: 'announcement-bar-1', // increment on change
```

Comment:
Increment on change 🙂 This will ensure that anyone who closed the banner and has the cookie will still see the new banner.
```suggestion
      id: 'announcement-bar-2', // increment on change
```